app/Models/Extensions/PhotoCast.php

```diff
@@ -15,6 +16,40 @@ trait PhotoCast
 	 */
 	public function toReturnArray(): array
 	{
+		if (strpos($this->type, 'video') === 0) {
```

Comment:
use `->is_video()` instead (there should be a function like that in the code base).
the `type` contains the mimetype data of the *photo* and there is unfortunately 1 exception.

Reply:
Great, I'll fix that. I tried to stay at close at the old code as possible and the old code did it that way. But of course, using `isVideo` is much cleaner. I'll update this PR.

Reply:
Fixed in commit c513d92d.